From b82a46e83caf00636160c7cc84a5cb9031262004 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 10 Mar 2023 11:56:10 +0000
Subject: [PATCH] ARM: dts: bcm2711-rpi-400: Add dummy cam1 regulator

Adding the cam1_reg label to the dummy camera regulator gets rid of an
error when all overlays are tried against all boards, and brings it
closer to what's happening in 6.2.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi-400.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-400.dts b/arch/arm/boot/dts/bcm2711-rpi-400.dts
index cfb335180ce7..b276f2d12af6 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-400.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-400.dts
@@ -537,6 +537,9 @@ &vchiq {
 	pinctrl-0 = <&audio_pins>;
 };
 
+cam0_reg: &cam_dummy_reg {
+};
+
 &genet_mdio {
 	clock-frequency = <1950000>;
 };
-- 
2.35.5

